src/lib/errors/docker-image-not-found-error.ts

```diff
@@ -0,0 +1,12 @@
+import { CustomError } from './custom-error';
+
+export class DockerImageNotFoundError extends CustomError {
+  private static ERROR_CODE = 404;
+
+  constructor(image: string) {
+    const message = `Failed to scan image ${image}. Please make sure the image and/or repository exist.`;
```

Comment:
I'd consider wrapping the image name in quotes to make it clear in the error message:

```suggestion
    const message = `Failed to scan image "${image}". Please make sure the image and/or repository exist.`;
```

Reply:
Thank you for taking your time to provide feedback 🙏 

I shall forward on your suggestion to the product owner and make adjustments if they agree. 👍

Reply:
The produce owner agree, I have implemented this suggestion, thank you 😄